checklists/alz_checklist.en.json

```diff
@@ -902,12 +888,11 @@
         {
             "category": "Network Topology and Connectivity",
             "subcategory": "Internet",
-            "text": "Deploy WAFs and other reverse proxies are required for inbound HTTP/S connections, deploy them within a landing-zone virtual network and together with the apps that they're protecting and exposing to the internet.",
+            "text": "When WAFs and other reverse proxies are required for inbound HTTP/S connections, deploy them within a landing-zone virtual network and together with the apps that they're protecting and exposing to the internet.",
```

Comment:
Can we consolidate this with D01.02?

Reply:
Removing D1.02 - I don't think any text is really needed here to "update"